### DISCUSSION TITLE
icon identification for LORAS

### DISCUSSION THREAD

**Discussion by hjmeneses** [Acly/krita-ai-diffusion#1468] [Ideas]
is it  possible to add a identification icon like the checkpoints?

**Comment by Acly** [comment]
You mean which base model (SD1.5/SDXL/...) they are meant for?

It's virtually impossible to detect because there are dozens of Lora formats and no good way to classify them (at least before loading everything).

Only way I can think of would be doing it manually (tedious) or maybe query civitai (not sure if they support that).

**Reply by hjmeneses** [reply]
Maybe just the ones with the base model mentioned in the inner json?

**Reply by hjmeneses** [reply]
Or load them and them keep a record so in the future we could tell

**Reply by hjmeneses** [reply]
Btw it seems vivir ai can be queried

https://github.com/xypher7/lora-metadata-viewer
